### ISSUE TITLE
Error: Unknown layer: Normalization

### ISSUE THREAD

**Issue by DobryninVyacheslav** [tensorflow/tfjs#4626]
Hello!

I trained a model in python and converted it for tfjs.
In the model I used a normalization layer:
```
from tensorflow.keras.layers.experimental import preprocessing

def build_and_compile_model(norm):
  model = keras.Sequential([
      norm,
      layers.Dense(64, activation='relu'),
      layers.Dense(64, activation='relu'),
      layers.Dense(1)
  ])

  model.compile(loss='mean_absolute_error',
                optimizer=tf.keras.optimizers.Adam(0.001))
  return model

normalizer = preprocessing.Normalization()
normalizer.adapt(np.array(train_features))

dnn_model = build_and_compile_model(normalizer)
```
After I run this model in js, I get an error:

> Uncaught (in promise) Error: Unknown layer: Normalization. This may be due to one of the following reasons:
> 1. The layer is defined in Python, in which case it needs to be ported to TensorFlow.js or your JavaScript code.

Please tell me:
do you need to add this class to tfjs?
or am I doing something wrong?

**Comment by rthadur** [comment]
Normalization has been added here https://github.com/tensorflow/tfjs/pull/2114 , which version of tfjs you are using ?

**Comment by pkosic** [comment]
I'm getting the same error (and my code is for all intents and purposes identical).

I'm getting the error with tfjs versions 3.0.0 and with 2.8.5.

The dense layers translate fine (tested with normalization layer removed) but there's something about the experimental.preprocessing normalization layer that tfjs appears to not like...

**Comment by rthadur** [comment]
cc @caisq

**Comment by pyu10055** [comment]
@DobryninVyacheslav We have support for [BatchNormalization](https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization) layer, but not the [preprocessing.Normalization](https://www.tensorflow.org/api_docs/python/tf/keras/layers/experimental/preprocessing/Normalization?hl=en) layer.

You can probably remove the layer from the model and use a set of tf ops to replace that in both python and tfjs:
```
tf.div(tf.sub(inputs, mean) / tf.sqrt(variance))
```

**Comment by pkosic** [comment]
I was going to do the manual normalization because I got scared of the API for BatchNormalization the first time I looked at it, but you gave the courage to use BN and all works - thanks! :)

A tangentially related question - running the model straight from the tfjs converter results in a typecasting error ((1) below). Manually changing the dtype of the first layer from float64 to float32 ((2) --> (3) below) fixes it and allows the model to function as expected. Is this manual workaround due to user error or due to a bug?

Thank you again!

(1) Uncaught (in promise) Error: The dtype of the feed (float32) can not be cast to the dtype of the key 'batch_normalization_3_input' (float64).
(2) {"batch_input_shape": [null, 4], "dtype": "float64", "sparse": false, "ragged": false, "name": "batch_normalization_3_input"}}
(3) {"batch_input_shape": [null, 4], "dtype": "float32", "sparse": false, "ragged": false, "name": "batch_normalization_3_input"}}
